publish/publish.proj

```diff
@@ -157,10 +157,11 @@
     </PropertyGroup>
     <MakeDir Directories="$(DownloadDirectory)"
              Condition="!Exists('$(DownloadDirectory)')" />
-    <DownloadBlobsFromAzure AccountName="$(AzureAccountName)"
+    <DownloadFromAzure AccountName="$(AzureAccountName)"
                            AccountKey="$(AzureAccessToken)"
                            ContainerName="$(ContainerName)"
                            BlobNames="@(_CoreHostPackages)"
+                           BlobNamePrefix="Runtime/$(SharedFrameworkNugetVersion)"
```

Comment:
Why is this prefix necessary when we list the exact files we want to download in `BlobNames`?